src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +184,23 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    public void verifyInstructorChangesAfterEdit(int instrNum, String newName, String newEmail,
+            Boolean newIsDisplayedToStudents, String newDisplayName, String newRole) {
+        assertEquals(newName, getInstructorName(instrNum));
+        if (!newEmail.isEmpty()) {
+            assertEquals(newEmail, getInstructorEmail(instrNum));
+        }
```

Comment:
Wait, ignore that last comment, my mistake. I thought this comment was about the `if (getEmailField(instrNum).getAttribute("readonly") == null)` in the same file :p) I will assert the same unchanged value in the case of the yet-to-join instructor.